### PR TITLE
added new app 'Instabot'

### PR DESCRIPTION
**Pull request by jngnyc**
Added a new app - [Instabot](instabot.io)

Instabot is a conversion chatbot for small to medium size businesses that understands your users' needs, provides information, and motivates them to convert quickly. Proven to increase conversions.

Here are 10 companies across the world and across all verticals/industries that currently love our product!
- https://www.knightscope.com/
- https://www.dentolo.de
- https://www.navycollege.navy.mil/
- https://njhotelsnearnyc.com
- https://www.tsishipping.com
- https://www.pagefreezer.com/
- https://blingby.com/
- https://www.donordrive.com/
- https://www.teamjackfoundation.org/
- https://www.bluefin.com/products/p2pe-point-to-point-encryption/

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -4629,6 +4629,21 @@
...
+				10,
+				32,
+				52,
+				58
+			 ],
+			 "icon": "Instabot.png",
+			 "js": { 
+				 "Instabot": ""
+			 },
+			 "script": "rokoInstabot\\.js",
```
[the remark this answers is not in the document]
Comment (jngnyc) [reply src/apps.json]: @jvoisin - fixed! Thanks for the suggestion!

**Comment by jngnyc** [comment]
@jvoisin - polite ping that I updated my PR to reflect your syntax suggestion - hoping for your PR approval! thanks and have a great weekend!

**Comment by jvoisin** [comment]
I don't have the commit bit, only @AliasIO can merge PR, and he's usually doing it every other month.

**Comment by AliasIO** [comment]
Thanks!

**Comment by jngnyc** [comment]
thanks @AliasIO !! Excited about the release!!

**Pull request merged**